engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -183,6 +183,16 @@ namespace dmGameSystem
         FillAttribute(gui_world->m_ParticleAttributeInfos.m_Infos[2], VERTEX_STREAM_COLOR,      dmGraphics::VertexAttribute::SEMANTIC_TYPE_COLOR,      4);
         FillAttribute(gui_world->m_ParticleAttributeInfos.m_Infos[2], VERTEX_STREAM_PAGE_INDEX, dmGraphics::VertexAttribute::SEMANTIC_TYPE_PAGE_INDEX, 1);
 
+        // Another way would be to use the vertex declaration, but that currently doesn't have an api
```

Comment:
Moved the code closer to the actual box vertex format, to keep them in sync more easily.